README.md

```diff
@@ -189,10 +189,17 @@ of the main ORT configuration file. It consists of sections related to different
 of these sections and the properties they can contain is described together with the corresponding sub components.
 
 While the file is rather static, there are means to override configuration options for a specific run of ORT or to
-customize the configuration to a specific environment. The following options are supported:
-
-* You can override the values of properties on the command line using the `-P` option. The option expects a
-  key-value pair. The key must define the full path to the property to be overridden, e.g.
+customize the configuration to a specific environment. The following options are supported whereas the priority is in the
```

Comment:
Commit message nits: Drop ".md" from "README.md". The "Closes #3242." line should come between body and signed-off footer.

Reply:
How about "The following options are supported, in order of precedence:"